problemsets-mat223/linearalbegra.tex

```diff
@@ -3084,10 +3084,87 @@ \section*{Task 2.3: Pat and Jamie}
 	and let $\mathcal R$ be rotation counter-clockwise by $90^\circ$.
 	\begin{parts}
 		\item Find a matrix $P$ so that $P\vec x=\mathcal P(\vec x)$ for all $\vec x\in\R^2$.
+			\begin{solution}
+				$P = \frac{1}{13}\mat{4&6\\6&9}$ is such a matrix.
+
+				We know from an earlier exercise that 
+				$\mathcal P(\vec x)=\dfrac{\vec x\cdot\vec u}{\vec u\cdot\vec u}\vec u$.
+				So if $\Set{\xhat,\yhat}$ is the standard basis for $\R^2$,
+				we can compute
+				\[
+					\mathcal P(\xhat)=\frac{2}{13}\vec u=\frac{1}{13}\mat{4\\6}
+					\qquad \text{and} \qquad
+					\mathcal P(\yhat)=\frac{3}{13}\vec u=\frac{1}{13}\mat{6\\9}.
+				\]
+				So since we know $\mathcal P$ is linear, for any $\vec x=\mat{x\\y}$,
```

Comment:
I'd like you to rephrase this as:

Suppose P is the matrix corresponding to \mathcal P. For any matrix, multiplying by $\xhat$ gives
the first column and multiplying by $\yhat$ gives the second. Therefore the first column of P is P\xhat=... and
the second column is P\yhat=...

Similarly for the others too, please. I'd like to emphasize the standard reasoning which is "when you don't know, make a matrix of unknowns and find a system of equations", which gives students something to grasp onto when they're lost.